### PR TITLE
fix(sbb-flip-card): fix background and height transitions

### PR DESCRIPTION
**Pull request by dauriamarco**
Implement correct animations for the `flip-card`.

### CODE REVIEW

File: src/elements/flip-card/flip-card/flip-card.scss [hunk 1]
```diff
@@ -14,11 +13,7 @@
     var(--sbb-animation-duration-5x)
   );
   --sbb-flip-card-summary-transition-delay: var(--sbb-animation-duration-2x);
-  --sbb-flip-card-details-transition-duration: var(
-    --sbb-disable-animation-zero-time,
-    var(--sbb-animation-duration-4x)
-  );
-  --sbb-flip-card-details-transition-delay: 0s;
+  --sbb-flip-card-details-transition-duration: var(--sbb-animation-duration-4x);
```
Comment (jeripeierSBB) [inline src/elements/flip-card/flip-card/flip-card.scss:16]: Please re-include the zero animation timing

File: src/elements/flip-card/flip-card/flip-card.scss [hunk 2]
```diff
@@ -35,14 +30,9 @@
 }
 
 :host([data-flipped]) {
-  --sbb-flip-card-background-color: var(--sbb-color-midnight);
-  --sbb-flip-card-details-transition-duration: var(
-    --sbb-disable-animation-zero-time,
-    var(--sbb-animation-duration-2x)
-  );
-  --sbb-flip-card-details-transition-delay: var(--sbb-animation-duration-5x);
+  --sbb-flip-card-details-transition-duration: var(--sbb-animation-duration-2x);
```
Comment (jeripeierSBB) [inline src/elements/flip-card/flip-card/flip-card.scss:33]: Please re-include the zero animation timing

File: src/elements/flip-card/flip-card/flip-card.ts [hunk 3]
```diff
@@ -58,9 +59,11 @@ export class SbbFlipCardElement extends SbbHydrationMixin(LitElement) {
 
   private _abort = new SbbConnectedAbortController(this);
   private _language = new SbbLanguageController(this);
+  private _flipCardResizeObserver = new AgnosticResizeObserver(() => this._setFlipCardHeight());
 
   public override connectedCallback(): void {
     super.connectedCallback();
+    this._flipCardResizeObserver.observe(this);
```
Comment (jeripeierSBB) [inline src/elements/flip-card/flip-card/flip-card.ts:66]: is disconnecting on disconnection not needed?

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]

**Review by jeripeierSBB — APPROVED** [review]
LGTM, thank you

File: src/elements/flip-card/flip-card/flip-card.scss [hunk 4]
```diff
@@ -108,8 +109,8 @@
     background-color: var(--sbb-color-midnight);
     width: var(--sbb-size-element-xs);
     height: var(--sbb-size-element-xs);
-    border-radius: var(--sbb-size-element-xs);
-    transition: var(--sbb-disable-animation-zero-time, var(--sbb-animation-duration-5x)) ease-out;
+    border-radius: #{sbb.px-to-rem-build(256)};
```
Comment (jeripeierSBB) [inline src/elements/flip-card/flip-card/flip-card.scss:112]: Please comment why the number of 256 pixels.

File: src/elements/flip-card/flip-card-summary/flip-card-summary.scss [hunk 5]
```diff
@@ -42,7 +42,7 @@
   padding-block: var(--sbb-spacing-responsive-s) var(--sbb-spacing-responsive-xs);
   margin: 0;
   grid-area: 1 / 1 / 2 / 2;
-  transform: translateY(var(--sbb-flip-card-title-translate-y-hover, #{sbb.px-to-rem-build(0)}));
+  transform: translateY(var(--sbb-flip-card-translate-title-y-hover, #{sbb.px-to-rem-build(0)}));
```
Comment (jeripeierSBB) [inline src/elements/flip-card/flip-card-summary/flip-card-summary.scss:45]: I think this needs to be applied to all the slotted content of the summary and not only the title, although we normally only expect titles slotted. But consumers could also slot `<h5>` or whatever that should receive the hover effect too.

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]
Almost done :-)

File: src/elements/flip-card/flip-card-summary/flip-card-summary.scss [hunk 6]
```diff
@@ -42,6 +42,9 @@
   padding-block: var(--sbb-spacing-responsive-s) var(--sbb-spacing-responsive-xs);
```
Comment (jeripeierSBB) [inline src/elements/flip-card/flip-card-summary/flip-card-summary.scss:42]: Somehow the padding of the flipped state changed. On purpose?
https://lyne-components-visual-regression-diff-pr3001.app.sbb.ch/compare/sbb-flip-card/flipped

Comment (dauriamarco) [reply src/elements/flip-card/flip-card-summary/flip-card-summary.scss:42]: I think the previous one was wrong 🤔

Comment (jeripeierSBB) [reply src/elements/flip-card/flip-card-summary/flip-card-summary.scss]: Then it should be ok :)

**Review by jeripeierSBB — APPROVED** [review]
LGTM, thanks! let's see if the grid alternative works as well

File: src/elements/flip-card/flip-card/flip-card.ts [hunk 7]
```diff
@@ -75,10 +75,15 @@ export class SbbFlipCardElement extends SbbHydrationMixin(LitElement) {
     );
   }
 
+  public override disconnectedCallback(): void {
+    super.disconnectedCallback();
+  }
```
Comment (jeripeierSBB) [inline src/elements/flip-card/flip-card/flip-card.ts:80]: Can be removed
```suggestion
```

**Review by jeripeierSBB — APPROVED** [review]
LGTM! Please remove one unneeded code block

**Pull request merged**